.ci.yaml

```diff
@@ -128,6 +128,7 @@ targets:
 
   ### Android tasks ###
   - name: Linux_android android_build_all_packages master
+    bringup: true # New target
```

Comment:
We might not be allowed to make `bringup: true` after the task has been moved out of `bringup` once :/ I guess the ci.yaml validation will tell!